### PR TITLE
Updates to phenotype hive import

### PR DESCRIPTION
**Pull request by ima23**
Changes:
- add species to OMIA import
- update mouse import for e101
- update code to address multiple jira tickets: ENSVAR-2494,ENSVAR-2870,ENSVAR-2738, ENSVAR-2885, ENSVAR-2872,ENSVAR-2494, ENSVAR-2479

### CODE REVIEW

File: modules/Bio/EnsEMBL/Variation/Pipeline/PhenotypeAnnotation/ImportHuman.pm [hunk 1]
```diff
@@ -54,9 +54,15 @@ sub fetch_input {
 
     unless ($run_type eq NONE) {
       my %import_species = SPECIES;
-      $run_type = GWAS if $run_type eq HUMAN; #GWAS analysis will trigger all the subsequent ones
-      $self->param('output_ids',  [ map { {species => $_} } @{$import_species{$run_type}} ]);
-      $self->print_pipelogFH("Setting up for $run_type import: ". join(", ",@{$import_species{$run_type}}). "\n") if $self->param('debug_mode') ;
+      if ( $run_type eq HUMAN ) {
+        #if HUMAN runtype, then select species by looking up MIMMORBID species
+        # expectation is that MIMMORBID will always be only homo_sapiens
+        $self->param('output_ids',  [ map { {species => $_} } @{$import_species{MIMMORBID}} ]);
```
Comment (aparton) [inline modules/Bio/EnsEMBL/Variation/Pipeline/PhenotypeAnnotation/ImportHuman.pm:60]: Does this need an if-else statement? Can you not do the same thing by setting `$var = ($run_type eq HUMAN) ? 'MIMMORBID' : $run_type;` and then remove the line duplication?

File: modules/Bio/EnsEMBL/Variation/Pipeline/PhenotypeAnnotation/BasePhenotypeAnnotation.pm [hunk 2]
```diff
@@ -131,10 +131,8 @@ sub get_pubmed_prefix {
 
 =head2 core_db_adaptor
 
-  Arg [1]    : Bio::EnsEMBL::DBSQL::DBAdaptor $db_adaptor (optional)
```
Comment (aparton) [inline modules/Bio/EnsEMBL/Variation/Pipeline/PhenotypeAnnotation/BasePhenotypeAnnotation.pm:134]: I know I'm being neurotic here, however the (optional) parameter is no longer relevant and should be removed

Comment (ima23) [reply modules/Bio/EnsEMBL/Variation/Pipeline/PhenotypeAnnotation/BasePhenotypeAnnotation.pm:134]: I agree, this line is already removed.

**Comment by aparton** [comment]
Merged into release/101 and master

**Pull request merged**